### DISCUSSION TITLE
Cloudflare Workers support

### DISCUSSION THREAD

**Discussion by nathanclevenger** [nanojsx/nano#108] [Help / Questions] [answered]
Has anyone tried getting nanojsx to be compatible with Cloudflare Workers?  Since it works on Deno I'm sure it's possible, but very curious to see if anyone has tried this yet.

**Comment by nathanclevenger** [comment]
Any thoughts on this @yandeu?

**Reply by yandeu** [reply]
Well, I haven't tried it, but NanoJSX works on Node.js and Deno. So it should work.

But, are you sure Cloudflare Workers uses Deno? I know it uses Node.js but don't know anything about Deno.

**Reply by nathanclevenger** [reply]
So Cloudflare Workers are actually not Node.js or Deno - they are Service Worker isolates on a V8 runtime, so that runtime actually is more similar to client-side JS in Chrome than Node ...  but I'll try to see if I can easily make it work this weekend, or what issues I run into.

**Comment by ArnabXD** [comment, marked as answer]
I have used `nano-jsx` with [hono](https://github.com/honojs/hono) and it works fine.

Offical example by hono - https://github.com/honojs/examples/tree/master/jsx-ssr

**Reply by nathanclevenger** [reply]
Thanks @ArnabXD!!!  My first attempt didn't seem to work, but this appears to be exactly what I was looking/hoping for!
